src/avfoundation.cs

```diff
@@ -7338,6 +7699,35 @@ interface AVVideoCompositionCoreAnimationTool {
 		AVVideoCompositionCoreAnimationTool FromComposedVideoFrames (CALayer [] videoLayers, CALayer inAnimationlayer);
 	}
 
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVCameraCalibrationData
+	{
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		[Export ("intrinsicMatrix")]
+		Matrix3 GetIntrinsicMatrix ();
+
+		[Export ("intrinsicMatrixReferenceDimensions")]
+		CGSize IntrinsicMatrixReferenceDimensions { get; }
+
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		[Export ("extrinsicMatrix")]
+		Matrix4 GetExtrinsicMatrix ();// should be a matrix 4x3
```

Comment:
The comment is a curious one, the matrix has to be a 4x3 but we do not have such a type since all matrices squared, yet this type must have the last row set to 0000, node how we could impose that.